lib/faktory_worker/job.ex

```diff
@@ -128,37 +125,9 @@ defmodule FaktoryWorker.Job do
 
   @doc false
   def perform_async(payload, opts) do
-    case Keyword.get(opts, :skip_pipeline, false) do
-      false ->
-        opts
-        |> push_pipeline_name()
-        |> perform_async(payload, opts)
-
-      true ->
-        opts
-        |> faktory_name()
-        |> push(payload)
-    end
-  end
-
-  @doc false
-  def perform_async(_, {:error, _} = error, _), do: error
-
-  def perform_async(pipeline_name, payload, opts) do
-    if Sandbox.active?() do
```

Comment:
I think we want to keep the sandbox, it is what we use to simulate the server during testing.

Reply:
@jeremyowensboggs makes sense, I overdid it with simplifications